fgd_cc_contacts/fgd_cc_contacts.py

```diff
@@ -0,0 +1,93 @@
+                                                 "manual verification and coding")
+    parser.add_argument("user", help="User launching this program, for use by TracedData Metadata")
+    parser.add_argument("phone_uuid_table_path", metavar="phone-uuid-table-path",
+                        help="JSON file containing an existing phone number <-> UUID lookup table. ")
+    parser.add_argument("fgd_cc_input_path", metavar="fgd-cc-input-path",
+                        help="Path to input file, containing a list of serialized TracedData objects as JSON")
+    parser.add_argument("demog_surveys_input_path", metavar="demog-surveys-input-path",
+                        help="Path to input file, containing a list of serialized TracedData objects as JSON")
+    parser.add_argument("json_output_path", metavar="json-output-path",
+                        help="Path to a JSON file to write processed TracedData messages to")
+    parser.add_argument("contacts_csv_output_path", metavar="contacts-csv-output-path",
+                        help="Path to a CSV file to write FGD/CC contacts to")
+
+    args = parser.parse_args()
+    user = args.user
+    phone_uuid_table_path = args.phone_uuid_table_path
+    fgd_cc_input_path = args.fgd_cc_input_path
+    demog_surveys_input_path = args.demog_surveys_input_path
+    json_output_path = args.json_output_path
+    contacts_csv_output_path = args.contacts_csv_output_path
+
+    # Load FGD/CC survey responses
+    with open(fgd_cc_input_path, "r") as f:
+        fgd_cc_data = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+
+    # Load coded demog surveys
+    with open(demog_surveys_input_path, "r") as f:
+        surveys = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+
+    # Filter out people who haven't answered the fgd_cc consent question
+    fgd_cc_consent_key = "Response_1 (Category) - wt_fgd_cc"
+    fgd_cc_data = [td for td in fgd_cc_data if fgd_cc_consent_key in td]
+
+    # Apply the demog surveys to the fgd_cc data
+    TracedData.update_iterable(user, "avf_phone_id", fgd_cc_data, surveys, "surveys")
+
+    # Annotate fgd_cc_data with whether or not the respondent's district is Mogadishu
+    raw_district_key = "District (Text) - wt_demog_1"
+    coded_district_key = "District (Text) - wt_demog_1_coded"
+    raw_age_key = "Age (Text) - wt_demog_2"
+    coded_age_key = "Age (Text) - wt_demog_2_coded"
+    for td in fgd_cc_data:
+        if raw_district_key not in td:
+            td.append_data({
+                raw_district_key: "NA"
+            }, Metadata(user, Metadata.get_call_location(), time.time()))
+
+        if coded_district_key not in td:
+            td.append_data({
+                coded_district_key: "none"
+            }, Metadata(user, Metadata.get_call_location(), time.time()))
+
+        is_mogadishu = td[coded_district_key] == "mogadishu" or td[coded_district_key] == "mogadisho"
```

Comment:
Same comments as with previous similar, I assume you'll address these at the same time